generators/app/templates/package.json

```diff
@@ -26,7 +26,8 @@
   },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react-dom": "^15.4.2",
+    "terra-base": "^0.x"
```

Comment:
So what should or shouldn't be a peer dependency? Should we have any dependencies? Additionally if we move react to a peer dependency, That means it's not installed automatically right? (This peer dependency stuff is confusing)